From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Fix build breakage in powerpc build

This fixes a problem with eeh_add_device_late() not being defined in the
ppc64 build process.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/asm-powerpc/eeh.h |    1 +
 1 files changed, 1 insertion(+)

--- linux-2.6.15.orig/include/asm-powerpc/eeh.h	2006-03-01 11:30:19.000000000 -0800
+++ linux-2.6.15/include/asm-powerpc/eeh.h	2006-03-01 12:04:25.000000000 -0800
@@ -61,6 +61,7 @@
  * to finish the eeh setup for this device.
  */
 void eeh_add_device_early(struct device_node *);
+void eeh_add_device_late(struct pci_dev *dev);
 void eeh_add_device_tree_early(struct device_node *);
 void eeh_add_device_tree_late(struct pci_bus *);
 
